### PR TITLE
LPS-99394

### PR DESCRIPTION
**Pull request by mateomustapic**
Hey @carloslancha 

Please review

Testing
------------------------------------------------------------------------------
Control Panel / Sites / Sites 
Select any Site dropdown menu

### CODE REVIEW

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld [hunk 1]
```diff
@@ -2430,7 +2436,7 @@
 		</attribute>
 		<attribute>
 			<description></description>
-			<name>userColorCssClass</name>
+			<name>userColorClass</name>
```
Comment (carloslancha) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld]: Why this change?

Comment (mateomustapic) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld]: Oh that is a leftover from other task. You can ignore and discard that change.

Comment (carloslancha) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld]: Nice

**Comment by carloslancha** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/80872. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Pull request submitted to https://github.com/carloslancha/liferay-portal/pull/218. [See changes here.](https://github.com/carloslancha/liferay-portal/compare/687e16d20f15~1...687e16d20f15) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/80920. [See changes here.](https://github.com/carloslancha/liferay-portal/compare/687e16d20f15~1...687e16d20f15) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Pull request submitted to https://github.com/jbalsas/liferay-portal/pull/2039. [See changes here.](https://github.com/carloslancha/liferay-portal/compare/687e16d20f15~1...687e16d20f15) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
